R/brood_check.R

```diff
@@ -1350,65 +1400,114 @@ compare_species_parents <- function(Brood_data, Individual_data, approved_list)
 compare_species_brood_parents <- function(Brood_data, Individual_data, approved_list) {
 
   # Find species information of mothers
-  Females <- Brood_data %>%
-    dplyr::filter(!is.na(FemaleID) & !is.na(MaleID)) %>%
-    dplyr::select(Row, PopID, BroodID, FemaleID, BroodSpecies = Species) %>%
+  females <- Brood_data %>%
+    dplyr::filter(!is.na(.data$FemaleID) & !is.na(.data$MaleID)) %>%
+    dplyr::select(.data$Row, .data$PopID, .data$BroodID,
+                  .data$FemaleID, "BroodSpecies" = .data$Species) %>%
     dplyr::left_join(Individual_data[,c("IndvID", "Species")],
                      by=c("FemaleID" = "IndvID")) %>%
-    dplyr::rename(FemaleSpecies = Species)
+    dplyr::rename("FemaleSpecies" = .data$Species)
 
   # Find species information of fathers
-  Males <- Brood_data %>%
-    dplyr::filter(!is.na(FemaleID) & !is.na(MaleID)) %>%
-    dplyr::select(Row, PopID, BroodID, MaleID, BroodSpecies = Species) %>%
+  males <- Brood_data %>%
+    dplyr::filter(!is.na(.data$FemaleID) & !is.na(.data$MaleID)) %>%
+    dplyr::select(.data$Row, .data$PopID, .data$BroodID,
+                  .data$MaleID, "BroodSpecies" = .data$Species) %>%
     dplyr::left_join(Individual_data[,c("IndvID", "Species")],
                      by=c("MaleID" = "IndvID")) %>%
-    dplyr::rename(MaleSpecies = Species)
-
-  # No errors
-  err <- FALSE
-  #error_records <- tibble::tibble(Row = NA_character_)
-  error_output <- NULL
+    dplyr::rename("MaleSpecies" = .data$Species)
 
-  # Warnings
   # Select records where parents are not of the same species as the brood
-  Multi_species_broods <- dplyr::left_join(Females, Males,
+  multi_species_broods <- dplyr::left_join(females, males,
```

Comment:
Just for ease of reading the code, would be good to use different object names for these different cases